e2e/suite1/specs/exemplars.spec.ts

```diff
@@ -35,16 +35,16 @@ describe('Exemplars', () => {
   });
 
   it('should be able to navigate to configured data source', () => {
-    e2e().intercept('POST', '/api/v1/query_exemplars', {
+    e2e().intercept('POST', '**/api/v1/query_exemplars', {
       fixture: 'exemplars-query-response.json',
     });
-    e2e().intercept('POST', '/api/v1/query_range', {
+    e2e().intercept('POST', '**/api/v1/query_range', {
       fixture: 'prometheus-query-range-response.json',
     });
-    e2e().intercept('POST', '/api/v1/query', {
+    e2e().intercept('POST', '**/api/v1/query', {
       fixture: 'prometheus-query-response.json',
     });
-    e2e().intercept('POST', '/api/ds/query', {
+    e2e().intercept('POST', '**/api/ds/query', {
```

Comment:
@zoltanbedi did you read https://docs.cypress.io/guides/references/migration-guide#cy-intercept-changes? Not sure this affect these, just making sure as the order here seems to be the same :)

Reply:
yep I saw it, I think it won't effect it.

Reply:
👍